lib/schemaTranslations.schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
I don't believe the existing json schema package which we use supports draft 7.

Reply:
Lets make sure this schema is the same as in this theme-registry pr: https://github.com/bigcommerce/theme-registry/pull/1016

Reply:
@junedkazi Yeah, I checked. `jsonschema` package supports draft 4.
Thanks, I will fix it.